lib/navigation.ts

```diff
@@ -35,7 +36,12 @@ export const navigateToProductList = async function (page: Page, product: string
 
 // Navigates to list of Streams for Apache Kafka instances
 export const navigateToKafkaList = async function (page: Page) {
-  await navigateToProductList(page, 'Streams for Apache Kafka', 'Kafka Instances');
+  if (process.env.ENV_TYPE == 'stage-new-ui') {
+    await page.goto(config.startingPage + '/application-services/new-streams');
```

Comment:
shouldn't this be set in startingPage env var ? And just via `ENV_TYPE` change selectors?

Reply:
It broke the version checks in pipeline stages, since it uses the same env variable.

Reply:
so in general the problem is, that UI is the same for everything except streams, right?